File: clients/javascript/src/models/idempotencyGetGroupIn.ts
// this file is @generated





export interface IdempotencyGetGroupIn {
    name: string;
}

export const IdempotencyGetGroupInSerializer = {
    _fromJsonObject(object: any): IdempotencyGetGroupIn {
        return {
            name: object['name'],
            };
    },

    _toJsonObject(self: IdempotencyGetGroupIn): any {
        return {
            'name': self.name,
            };
    }
}

File: clients/rust/src/models/idempotency_get_group_in.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[non_exhaustive]
#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct IdempotencyGetGroupIn {
    pub name: String,
}

impl IdempotencyGetGroupIn {
    pub fn new(name: String) -> Self {
        Self { name }
    }
}
